src/sentry/tasks/weekly_escalating_forecast.py

```diff
@@ -60,16 +60,25 @@ def run_escalating_forecast() -> None:
 )
 @retry
 def generate_forecasts_for_projects(project_ids: List[int]) -> None:
-    for until_escalating_groups in chunked(
-        RangeQuerySetWrapper(
+
+    query_until_escalating_groups = [
+        group
+        for group in RangeQuerySetWrapper(
             Group.objects.filter(
                 status=GroupStatus.IGNORED,
                 substatus=GroupSubStatus.UNTIL_ESCALATING,
                 project_id__in=project_ids,
                 last_seen__gte=datetime.now() - timedelta(days=7),
-            ),
+            ).prefetch_related(
+                "project"
+            ),  # TODO: Remove this once the feature flag is removed
```

Comment:
I'd recommend `.select_related("project", "project__organization")`. `prefetch_related` makes an additional query per relation and isn't required here since this is just a fk lookup. You also need to include organization here since you're accessing that property on the project as well

Reply:
Thanks! I am not too familiar with these types of queries